lua/lvim/bootstrap.lua

```diff
@@ -3,6 +3,7 @@ local M = {}
 if vim.fn.has "nvim-0.8" ~= 1 then
   vim.notify("Please upgrade your Neovim base installation. Lunarvim requires v0.8+", vim.log.levels.WARN)
   vim.wait(5000, function()
+    ---@diagnostic disable-next-line: redundant-return-value
```

Comment:
you could also move these as well to that same PR

Reply:
all the `---@diagnostic` ?